modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -17,65 +17,105 @@ date_format = '%Y-%m-%d:%H:%M:%S'
 mongo_server = '<%= @server %>'
 key_fingerprint = '<%= @private_gpg_key_fingerprint %>'
 s3_bucket = '<%= @s3_bucket %>'
-standalone = '<%= @standalone %>'
+standalone = <%= @standalone %>
 start = datetime.now().strftime(date_format)
 start_time = time.time()
-file = 'mongodump-<%= @fqdn %>-' + start + '.tar.gz.gpg'
+today = datetime.today().strftime('%Y-%m-%d')
 ############################################
 
 def logger(str):
```

Comment:
I don't think this redirection is working - I'm getting stuff in `/var/mail/govuk-backups` when I run this in vagrant.